client/src/modules/stock/lots/registry.js

```diff
@@ -92,6 +92,40 @@ function StockLotsController(
       displayName : 'STOCK.EXPIRATION',
       headerCellFilter : 'translate',
     },
+    {
+      field : 'avg_consumption',
+      displayName : 'STOCK.CMM',
```

Comment:
Are these numbers?  If so, could we put `type: "number"` on the field?  It will make sure we sort these columns in the right way.